rootfs/api/settings/production.py

```diff
@@ -294,6 +294,29 @@
 # Has priority over DEIS_DEPLOY_PROCFILE_MISSING_REMOVE
 DEIS_DEPLOY_REJECT_IF_PROCFILE_MISSING = bool(strtobool(os.environ.get('DEIS_DEPLOY_REJECT_IF_PROCFILE_MISSING', 'false')))  # noqa
 
+# True, true, yes, y and more evaluate to True
+# False, false, no, n and more evaluate to False
+# https://docs.python.org/3/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool
+# see the above for all available options
+#
+# If a user deploys one build with a Sidecarfile but then forgets to in the next one
+# then let that go through without removing the sidecars for the processes
+#
+# If the user has a Sidecarfile in both deploys then sidecars are added / removed as per usual
+#
+# By default the sidecaress types are scaled down unless this setting is turned on
+DEIS_DEPLOY_SIDECARFILE_MISSING_REMOVE = bool(strtobool(os.environ.get('DEIS_DEPLOY_SIDECARFILE_MISSING_REMOVE', 'true')))  # noqa
```

Comment:
If I got it right, if I've made a deploy with a sidecarfile before and then make a new one without the sidecar (like a deis pull from outside the project's dir) then my sidecar will be deleted by default, is that right? If so, shouldn't we turn this off by default?

Reply:
Yes, that's exactly what happens. I'll change it to `false` because I also think it is a better default value.

Reply:
however, if we change this to false, how would I remove a sidecar from a process?

Reply:
You would need to provide a Sidecarfile with empty sidecars for each process.

Reply:
Just for reference, I was wrong. We cannot supply a Sidecarfile with empty sidecars.

This file

```yaml
cmd:
```

results in 

```
remote: 2018/03/16 12:55:18 Error running git receive hook [The controller returned an error when publishing the release: Unknown Error (400): {"sidecarfile":["Sidecar config can't be empty for process type"]}]
```

However if I use an invalid process I am able to remove the sidecar from the cmd one. Example Sidecarfile:
```
nope:
- name: busybox
```

I know, this is damn ugly because we'll need a dummy Sidecarfile that actually does nothing. We'll probably need a command to remove the Sidecarfile, maybe a flag that allows removing sidecars, but for now we can achieve this by turning the flag `DEIS_DEPLOY_SIDECARFILE_MISSING_REMOVE` on and deploying a new version without the sidecar.